--- rules.mk.org	2006-11-07 22:52:22.000000000 +0100
+++ rules.mk	2006-11-07 22:51:28.000000000 +0100
@@ -882,7 +882,7 @@
 ifneq (,$(filter-out OpenVMS OS2 WIN%,$(OS_TARGET)))
 # Can't use sed because of its 4000-char line length limit, so resort to perl
 .DEFAULT:
-	@perl -e '                                                            \
+	@perl -e "                                                            \
 	    open(MD, "< $(DEPENDENCIES)");                                    \
 	    while (<MD>) {                                                    \
 		if (m@ \.*/*$< @) {                                           \
@@ -909,7 +909,7 @@
 	    } elsif ("$<" ne "$(DEPENDENCIES)") {                             \
 		print "$(MAKE): *** No rule to make target $<.  Stop.\n";     \
 		exit(1);                                                      \
-	    }'
+	    }"
 endif
 
 #############################################################################
